ztp/argocd/Containerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/openshift/origin-cli:latest
+
+USER root
+
+# For CLI use in pre-sync
+RUN INSTALL_PKGS="tar unzip golang jq" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    yum clean all -y && \
+    curl -sSLo - https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v4.1.3/kustomize_v4.1.3_linux_amd64.tar.gz | \
```

Comment:
Kustomize version embedded in the oc does not support what we need. 
We are already discussing ways to drop the use of kustomize inside of this container, so I think this will be the way forward for support.
@nishant-parekh